src/tests/suite/daos_iotest.h

```diff
@@ -40,6 +40,13 @@ extern int dts_obj_replica_cnt;
 #define IOREQ_SG_NR	5
 #define IOREQ_SG_IOD_NR	5
 
+#define MAX_EXT_NUM		5
+#define MAX_DISTANCE		10
+#define MAX_EXTENT_SIZE		50
+#define MAX_OFFSET		1048576
+#define SINGLE_REC_RATE		20
+#define MAX_EPOCH_TIMES		20
```

Comment:
originally those macros only used in one .c file, now you moved those to a common test header file so maybe better to add "DTS_" prefix to avoid possible naming conflict.